### PR TITLE
Issue Status fixes

### PR DESCRIPTION
**Pull request by haiodo**
Signed-off-by: Andrey Sobolev <haiodo@gmail.com>

# Contribution checklist

## Brief description

* Allow issue status per every grouping category
* Minor fixes for time reporting dialog.

image

Fixes to reporting dialog.
image

Estimation could be sed with clicking on estimation, inside popup.

Screenshot 2022-12-29 at 01 47 32

## Checklist

* [x] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [x] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [x] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [x] - Does new code is well documented ?

## Related issues

A list of closed updated issues

### CODE REVIEW

File: plugins/tracker-resources/src/components/issues/StatusEditor.svelte [hunk 1]
```diff
@@ -34,6 +34,9 @@
   export let justify: 'left' | 'center' = 'left'
   export let width: string | undefined = undefined
 
+  export let issues: Issue[] | undefined = undefined
```
Comment (BykhovDenis) [inline plugins/tracker-resources/src/components/issues/StatusEditor.svelte]: Why we need it?

File: plugins/tracker-resources/src/components/issues/StatusEditor.svelte [hunk 2]
```diff
@@ -34,6 +34,9 @@
...
+  export let groupBy: string | undefined = undefined
```
Comment (BykhovDenis) [inline plugins/tracker-resources/src/components/issues/StatusEditor.svelte]: Why we need it?

**Review by BykhovDenis — APPROVED** [review]

File: plugins/tracker-resources/src/components/issues/StatusEditor.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/tracker-resources/src/components/issues/StatusEditor.svelte]: Removed.

File: plugins/tracker-resources/src/components/issues/StatusEditor.svelte (hunk 1 above)
Comment (haiodo) [reply plugins/tracker-resources/src/components/issues/StatusEditor.svelte]: Removed.

**Pull request merged**